docs/asciidoc/modules/ROOT/partials/usage/apoc.load.directory.async.adoc

```diff
@@ -198,13 +216,22 @@ If we execute `call apoc.load.directory.async.list`, we obtain, for example:
 | `listenerName` | `ERROR` | `*.csv` | `'create (n:Node)'` | `'path'` | `{}` | `"org.neo4j.graphdb.QueryExecutionException: Failed to invoke procedure apoc.load.csv: Caused by: java.io.FileNotFoundException ....`
 |===
 
-=== Procedure enabling
+=== Configuration
 
-Please note that to use the `apoc.load.directory.async.*` procedures,
-we'll have to enable the following config option:
+Please note that to use the `apoc.load.directory.async.*` procedures, the following config needs to be enabled:
 
 .apoc.conf
 [source,properties]
 ----
 apoc.import.file.enabled=true
 ----
+
```

Comment:
@nadja-muller There were some text here before which you removed in your commit about listenEventType, I'm not sure whether that is covered anywhere else